templates/commands/render/action.go

```diff
@@ -141,6 +141,29 @@ func templateAndCompileRegexes(regexes []model.String, scope *common.Scope) ([]*
 	return compiled, merr
 }
 
+// processPaths processes a list of input String paths for go templating, relative paths,
+// OS-specific slashes, and (soon) file globbing.
+func processPaths(paths []model.String, sp *stepParams) ([]model.String, error) {
+	pathsCopy := make([]model.String, len(paths))
+	copy(pathsCopy, paths)
+
+	var err error
+	for i, p := range pathsCopy {
+		p.Val, err = parseAndExecuteGoTmpl(p.Pos, p.Val, sp.scope)
+		if err != nil {
+			return nil, err
+		}
+		p.Val, err = safeRelPath(p.Pos, p.Val)
+		if err != nil {
+			return nil, err
+		}
+		p.Val = filepath.FromSlash(p.Val)
```

Comment:
Hm I think FromSlash should happen before safeRelPath. I think we want safeRelPath to see the true local-OS-style path.

Reply:
Reordered.

Reply:
Reordering this seems to break tests for windows: https://github.com/abcxyz/abc/actions/runs/6190617112/job/16807205539?pr=208

Reply:
Reordered, fixed test.